scopes/generator/generator/new.cmd.ts

```diff
@@ -17,9 +17,12 @@ export class NewCmd implements Command {
     {
       name: 'template-name',
       description:
-        "the name of the workspace template (run 'bit templates', outside of a workspace, to get a list of available templates)",
+        "the name of the workspace template (run 'bit templates', outside of a workspace, to get a list of available workspace templates)",
```

Comment:
we should probably remove the commas inside the brackets.

Reply:
agreed, done